src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -44,16 +44,18 @@ public static CreditCardFund fund(String amount, CreditCardPriority primaryOrSec
                 break;
         }
 
-        if (cardID == null || cardID.isEmpty() || !cardID.startsWith("card_")) {
```

Comment:
IDE suggestion, if the it's an empty string (because it passed the first null check), then checking whether the string is empty is redundant over checking if the string starts with a specific substring.